lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -0,0 +1,85 @@
+import { createElement } from 'preact';
+import propTypes from 'prop-types';
+import { useEffect, useState } from 'preact/hooks';
+
+import StudentSelector from './StudentSelector';
+import updateClientConfig from '../utils/update-client-config';
+
+/**
+ * The LMS Grader menu that is fixed at the top of the page. This menu shows which assignment is currently
+ * active as well as a list of students to both view and submit grades for.
+ */
+
+export default function LMSGrader({
+  children,
+  students,
+  onChangeSelectedUser,
+}) {
+  // No initial current student selected
+  const [currentStudent, setCurrentStudent] = useState(-1);
+
+  useEffect(() => {
+    if (students[currentStudent]) {
+      updateClientConfig({
+        focus: {
+          user: {
+            username: students[currentStudent].userid,
+            displayName: students[currentStudent].displayName,
+          },
+        },
+      });
+    }
+  }, [students, currentStudent]);
+
+  /**
+   * Shows the current student index if a user is selected, or the
+   * total student count otherwise.
+   */
+  const renderStudentCount = () => {
+    if (currentStudent >= 0) {
+      return <span>{`${currentStudent + 1}/${students.length}`}</span>;
+    } else {
+      return <span>{`${students.length} students`}</span>;
+    }
+  };
+
+  /**
+   * Callback to set the current selected student and fires a callback
+   * to this parent with the new username.
+   */
+  const onSelectStudent = studentIndex => {
+    setCurrentStudent(studentIndex);
+    onChangeSelectedUser(students[studentIndex].userid);
+  };
+
+  return (
+    <div className="LMSGrader">
+      <header>
+        <ul className="LMSGrader_flex-container">
+          <li className="LMSGrader__course-assignment">
+            <h2>Course Name</h2>
```

Comment:
I wonder if we might pull these two elements for now (course, assignment name)—we haven't actually talked about how we'll populate them (it shouldn't be too tricky, but I'd hate to hold up rolling this out because we're hung up on displaying these values)...thoughts?

Reply:
Thoughts on this one? I notice that presently, there's no padding on the left here, either:

![image](https://user-images.githubusercontent.com/439947/66047148-465aff80-e4f5-11e9-9866-6c24d4e555d8.png)

I think it would be ideal just to not render these in this round of work—we can add back when we have values to show?